### PR TITLE
FR-664  [Storyboard] I want to hide old storyboard pictures on new commits of PR

### PR DESCRIPTION
**Pull request by heriniaina-rak**

### CODE REVIEW

**Review by ttoavina — APPROVED** [review]

File: lib/src/storyboard_controller.dart [hunk 1]
```diff
@@ -246,7 +249,7 @@ class StoryBoardController {
     final byteList = bytes.buffer.asUint8List();
     if (isCI()) {
       print(
-          "Storyboard: ${view.widget.graphForCiAuto?.relationDescription ?? "graphForCiAuto is null"}");
+          "Storyboard: ${view.widget.graphForCiAuto?.imageNumber} : ${view.widget.graphForCiAuto?.relationDescription ?? "graphForCiAuto is null"}");
```
Comment (andromalala) [inline lib/src/storyboard_controller.dart]: https://github.com/flitsoft/coding-standard/blob/master/CODING-STANDARDS.md#dont-add-logic-in-print-statement

**Review by ttoavina — APPROVED** [review]

File: lib/src/storyboard_controller.dart [hunk 2]
```diff
@@ -230,8 +230,11 @@ class StoryBoardController {
     final answer = await uploadTask;
     final url = await answer.ref.getDownloadURL();
     print("Download URL: $url");
+    await Future.delayed(Duration(seconds: 4));
     if (showFlowInPullRequest) {
       print("Download IMAGE PR: $url");
```
Comment (tolotrasamuel) [inline lib/src/storyboard_controller.dart]: atao mazava hoe special ito print ito, de atao anaty variable constant ilay key

Comment (tolotrasamuel) [reply lib/src/storyboard_controller.dart]: Download URL avadika => DownloadURLKeyNoShowInPR: 
Download URL avadika => DownloadURLKeyShowInPR:

Fa iray amreo iany no alefa

Comment (tolotrasamuel) [reply lib/src/storyboard_controller.dart]: conclusion, tokony print 1 ftsn, tsisy delay 4

**Pull request merged**